packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,74 @@ return [
 ];
 ```
 
+## Usage
+
+```php
+
+namespace App\Admin;
+
+use Runroom\SortableBehaviorBundle\Admin\AbstractSortableAdmin;
+
+class ExampleAdmin extends AbstractSortableAdmin
+{
+    // you can use SortableAdminTrait instead AbstractSortableAdmin
+
+    protected function configureListFields(ListMapper $list): void
+    {
+        $list
+            // ... some fields
+            ->add('_action', null, [
+                'actions' => [
+                    // ... some actions 
+                    'move'   => ['template' => '@RunroomSortableBehavior/sort.html.twig'],
+                ],
+            ]);
+    }
+}
+```
+
+### Configuration
+```yaml
+# app/config/config.yml
+runroom_sortable_behavior:
+    db_driver: orm # possible values: orm, mongodb 
+    position_field:
+        default: position
+        entities:
+            AppBundle\Entity\Foobar: order
+            AppBundle\Entity\Baz: rang
+    sortable_groups:
+        entities:
+            AppBundle\Entity\Baz: [ group ]
+            
+```
+
+### Use a draggable list instead of up/down buttons
+
+In order to use a draggable list instead of up/down buttons, change the template in the `move` action to `@RunroomSortableBehavior/sort.html.twig`.
```

Comment:
```suggestion
In order to use a draggable list instead of up/down buttons, change the template in the `move` action to `@RunroomSortableBehavior/sort_drag_drop.html.twig`.
```